### PR TITLE
[PW-4391]Update Readme based on the new template

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,61 @@
...
 
-To validate functionality of client, use `bundle install --with development` and `rspec` to run mock API tests.
+To validate functionality of client and run mock API tests use 
 
+~~~~bash  
+bundle install --with development 
+~~~~
+and
+~~~~bash 
+`rspec`
```
Comment (AlexandrosMor) [inline README.md:45]:
```suggestion
rspec
```

File: README.md [hunk 2]
```diff
@@ -1,42 +1,61 @@
...
-require 'adyen-ruby-api-library'
+### General use with API key
 
+~~~~bash 
+require 'adyen-ruby-api-library'
+~~~~
+~~~~ruby
 adyen = Adyen::Client.new
 
 # for API-key based implementations
```
Comment (AlexandrosMor) [inline README.md:61]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -45,10 +64,11 @@ adyen.api_key = 'AF5XXXXXXXXXXXXXXXXXXXX'
 # for basic-auth based implementations
```
Comment (AlexandrosMor) [inline README.md:64]:
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -65,126 +85,36 @@ response = adyen.checkout.payments({
...
+~~~~
 
-## Support
-If you have a feature request, or spotted a bug or a technical problem, create a GitHub issue. For other questions, contact our [support team](https://support.adyen.com/hc/en-us/requests/new?ticket_form_id=360000705420).
+### Example integration
+
+For a closer look at how our PHP library works, clone our [example integration](https://github.com/adyen-examples/adyen-rails-online-payments). This includes commented code, highlighting key features and concepts, and examples of API calls that can be made using the library.
+
+### Running the tests
+For the test cases you need the PCI permission enabled on you account. There are no test cases for CSE because credit card data is encrypted through our javascript library.
```
Comment (AlexandrosMor) [inline README.md:100]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -65,126 +85,36 @@ response = adyen.checkout.payments({
...
+By default the test will then be skipped. If you have these permissions fill in your account details in the config/test.ini file to let the test work.
```
Comment (AlexandrosMor) [inline README.md:101]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -65,126 +85,36 @@ response = adyen.checkout.payments({
...
+To make the automatic testing cases work for your account change the credentials in the config/test.ini file.
```
Comment (AlexandrosMor) [inline README.md:102]:
~~~~bash  
bundle install --with development 
~~~~
```suggestion
To run the tests use : 
~~~~bash  
bundle install --with development 
~~~~
```

**Review by msilvagarcia — APPROVED** [review]

**Review by crrood — COMMENTED** [review]
Hey @AlexandrosMor,

Looks like "PHP" got inserted for "Ruby" in the copy / pasting of the template.  Also, the punctuation was removed from the first sentence of the "installation" section.  Otherwise looks good.

File: README.md [hunk 7]
```diff
@@ -1,54 +1,72 @@
 # Adyen API Library for Ruby
 
-The Adyen API Library for Ruby lets you easily work with Adyen's API.
+
+This is the officially supported PHP library for using Adyen's APIs.
```
Comment (AlexandrosMor) [inline README.md:4]:
```suggestion
This is the officially supported ruby library for using Adyen's APIs.
```

**Comment by AlexandrosMor** [comment]
Hello @crrood, 

Thank you for checking, I corrected it :)

regards,

**Review by crrood — COMMENTED** [review]
Hey @AlexandrosMor,

Thanks for fixing the "PHP", but it looks like the "installation" section is still missing punctuation.  Can we change this sentence:
"The sole dependency is faraday for http communication run the following command which installs faraday if you don't already have it"
to
"The sole dependency is faraday for HTTP communication.  Run the following command which installs faraday if you don't already have it:"

File: README.md [hunk 8]
```diff
@@ -65,126 +83,37 @@ response = adyen.checkout.payments({
...
-**bin_lookup:**
-- get_3ds_availability
-- get_cost_estimate
+~~~~
 
-## Support
-If you have a feature request, or spotted a bug or a technical problem, create a GitHub issue. For other questions, contact our [support team](https://support.adyen.com/hc/en-us/requests/new?ticket_form_id=360000705420).
+### Example integration
+
+For a closer look at how our PHP library works, clone our [example integration](https://github.com/adyen-examples/adyen-rails-online-payments). This includes commented code, highlighting key features and concepts, and examples of API calls that can be made using the library.
```
Comment (msilvagarcia) [inline README.md:95]:
```suggestion
For a closer look at how our Ruby library works, clone our [example integration](https://github.com/adyen-examples/adyen-rails-online-payments). This includes commented code, highlighting key features and concepts, and examples of API calls that can be made using the library.
```

**Review by crrood — APPROVED** [review]

File: README.md [hunk 9]
```diff
@@ -1,54 +1,72 @@
...
 
-The Adyen API Library for Ruby lets you easily work with Adyen's API.
+
+This is the officially supported Ruby library for using Adyen's APIs.
 
 ## Integration
+The library supports all APIs under the following services:
 
-The Library supports all APIs under the following services:
+* [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v65/overview): Our latest integration for accepting online payments. Current supported version: **v65**
```
Comment (cyattilakiss) [inline README.md:9]: after the other PR it needs to be 67

Comment (AlexandrosMor) [inline README.md:9]:
```suggestion
* [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/overview): Our latest integration for accepting online payments. Current supported version: **v67**
```

Comment (AlexandrosMor) [reply README.md]: @cyattilakiss done 👍🏻

File: README.md [hunk 10]
```diff
@@ -65,126 +83,37 @@ response = adyen.checkout.payments({
...
   :merchantAccount => "YourMerchantAccount"
 })
-```
+~~~~
 
-### Change API Version
-```ruby
+- Change API Version
+~~~~ruby
 adyen.checkout.version = 65
```
Comment (cyattilakiss) [inline README.md:90]: This as well I think

```suggestion
adyen.checkout.version = 67
```

**Review by msilvagarcia — DISMISSED** [review]

**Review by crrood — DISMISSED** [review]

**Review by wboereboom — APPROVED** [review]
🥇

**Review by msilvagarcia — APPROVED** [review]

**Review by cyattilakiss — APPROVED** [review]

**Review by crrood — APPROVED** [review]

**Pull request merged**